easypost/easypost_object.py

```diff
@@ -135,32 +138,34 @@ def setdefault(self, k, default=None):
     def __setitem__(self, k, v):
         setattr(self, k, v)
 
-    def keys(self):
+    def keys(self) -> List:
```

Comment:
Where possible, can we specify what it's a list of? Let's try to avoid empty `Dict` and `List` types where possible and also specify what types go in them.